modules/local/simpleaf_quant.nf

```diff
@@ -1,11 +1,11 @@
-process SALMON_ALEVIN {
```

Comment:
Also this one could go to nf-core/modules?

Reply:
The same as the other.